src/main/java/net/dv8tion/jda/core/requests/WebSocketClient.java

```diff
@@ -1319,8 +1321,11 @@ private void setupHandlers()
         handlers.put("VOICE_STATE_UPDATE",          new VoiceStateUpdateHandler(api));
 
         // Unused events
-        handlers.put("CHANNEL_PINS_UPDATE",         new SocketHandler.NOPHandler(api));
-        handlers.put("WEBHOOKS_UPDATE",             new SocketHandler.NOPHandler(api));
+        final SocketHandler.NOPHandler nopHandler = new SocketHandler.NOPHandler(api);
```

Comment:
Technically, since `NOPHandler` will never be implemented, we should have it be a `public static final` instance in `SocketHandler` instead of in-line creating one here.

Reply:
The reason is that the SocketHandler constructor expects a JDA instance and I don't want to add confusing situations where that JDA instance isn't present/null

Reply:
Okay, I can get behind that.
However, lets create the final variable at the top of the method, not in the middle.